test_elasticsearch/test_server/test_rest_api_spec.py

```diff
@@ -275,6 +276,14 @@ def run_match(self, action):
             else:
                 assert expected == value, "%r does not match %r" % (value, expected)
 
+    def run_contains(self, action):
+        for path, expected in action.items():
+            value = self._lookup(path)  # list[dict[str,str]] is returned
+            expected = self._resolve(expected)  # dict[str, str]
+
+            if expected not in value:
+                raise AssertionError(f"{expected} does not match {value}")
```

Comment:
The assertation message here could be better, something like `"%s is not contained by %s" % (expected, value)`

Also we can't use f-strings yet for this project, we still support Python <3.6